project/AkkaBuild.scala

```diff
@@ -40,7 +40,7 @@ object AkkaBuild extends Build {
 
   val requestedScalaVersion = System.getProperty("akka.scalaVersion", "2.10.5")
   val Seq(scalaEpoch, scalaMajor) = """(\d+)\.(\d+)\..*""".r.unapplySeq(requestedScalaVersion).get.map(_.toInt)
-  val streamAndHttpVersion = "1.0-SNAPSHOT"
```

Comment:
Those changes came in from the releasing-akka-stream-and-http-experimental-1.0 branch. Should I rebase on release-2.3-dev?